setup.py

```diff
@@ -36,6 +36,7 @@
             "coverage[toml]",
             "filelock",
             "pytest-insta",
+            "vyper",
```

Comment:
We should pin this to 0.3.7

Reply:
In this case, we loose compatibility with Python 3.12+ : 

https://github.com/vyperlang/vyper/blob/6020b8bbf66b062d299d87bc7e4eddc4c9d1c157/setup.py#L92

Reply:
This is partly why we were using a binary so we aren't constrained by their packages/version

Reply:
I see! Lets close this PR and just update the wiki to precise that running the tests requires to install `vyper` (and `ganache`).